gpm_api/utils/geospatial.py

```diff
@@ -150,15 +145,17 @@ def get_extent(xr_obj, padding: Union[int, float, tuple, list] = 0):
         raise TypeError("Accepted padding type are int, float, list or tuple.")
     lon = xr_obj["lon"].data
     lat = xr_obj["lat"].data
+
+    if _is_crossing_dateline(lon):
+        raise NotImplementedError(
+            "The object cross the dateline. The extent can't be currently be defined."
```

Comment:
currently **be** defined ... remove be 
(I can't suggest code directly ... mystery...)